src/main/java/org/broadinstitute/consent/http/models/UserUpdateFields.java

```diff
@@ -177,6 +179,31 @@ public List<Integer> getRoleIdsToRemove(List<Integer> currentUserRoleIds) {
                   VALID_ROLE_IDS.contains(
                       id);                            // Only remove roles we know about
             })
-        .collect(Collectors.toList());
+        .toList();
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
This was only really needed for testing.

Reply:
That's good because since this is a mutable object the results aren't valid over time. It would be better to make this immutable using `record` so you don't have write and maintain these methods for tests. But that would require a bigger change.